### DISCUSSION TITLE
"" quotation marks in the Writing DAX Queries tutorial

### DISCUSSION THREAD

**Discussion by BobCreamer** [DaxStudio/DaxStudio#693] [Q&A]
An editing notes about the text.

Early on in the Writing DAX  Queries tutorial we are given:

CALCULATETABLE ( Customer, Customer[City] = “Redmond” )

For those of us copying text from the page and trying it out in DaxStudio we get a problem.
The double apostrophe quotation marks which DAX requires have somehow been re-formatted as curly marks in the text.
My copy paste attempt gets rejected by DaxStudio.

Not a big deal, but one of those early stumbling blocks that deter the less determined from making progress.
![Quotation marks](https://user-images.githubusercontent.com/41072468/127923273-03464907-2234-4bff-8499-9d916872f9be.jpg)

**Comment by BobCreamer** [comment]
In fact this happens throughout the text.  It might be the way my Chrome renders the text.
Of course I am a quick learner and know how to correct for it now.

**Comment by dgosbell** [comment]
Thanks for reporting this, it does appear to be something the browser is doing. I'll look at fixing this, maybe by converting the code blocks to just raw unformatted text, one of the main ideas was for people to be able to copy / paste and this does ruin that experience.
